test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -89,6 +96,75 @@ private void Consume(IEnumerable<IrisPrediction> predictions)
             return model;
         }
 
+        private static IPredictor TrainSentimentCore()
+        {
+            var dataPath = s_sentimentDataPath;
+            using (var env = new TlcEnvironment(seed: 1))
+            {
+                // Pipeline
+                var loader = new TextLoader(env,
+                new TextLoader.Arguments()
```

Comment:
Thanks! I also indented the subsequent lines since they are actually all inside one pair of parantheses, which may not be apparent from the current code style.